documentation/asciidoc/topics/proc_configuring_autoscaling.adoc

```diff
@@ -0,0 +1,30 @@
+[id='configuring_autoscale-{context}']
+= Configuring Automatic Scaling
+When you create clusters with {cacheservice} nodes, {ispn_operator} can
+automatically scale up or down based on memory usage.
+
+{ispn_operator} monitors memory usage for each node in the {brandname} cluster.
+If it detects that the {cacheservice} cluster requires additional capacity,
+{ispn_operator} creates new nodes rather than evicting entries. Likewise, if it detects that memory usage is below a certain threshold, {ispn_operator} shuts down nodes.
+
+.Procedure
+
+. Add the `spec.autoscale` resource to your **Infinispan** CR to enable automatic scaling.
+. Configure memory usage thresholds and number of replicas to control when {ispn_operator} automatically scales nodes up or down.
+. Apply the changes.
+
+----
+spec:
+  ...
+  autoscale: <1>
```

Comment:
@rigazilla as idea, maybe we can also add enable flag to the  _autoscale_ section as it contains several options. When user will try to playground with this options, it would be nice to easy enable/disable without add/remove spec section.

Reply:
@dmvolod sounds a good improvement, maybe we can add a `disable` flag so it's not showed by default

Reply:
@dmvolod and @rigazilla need a separate issue for the `disable` flag too?

Reply:
Yes, @rigazilla disable flag is much better
@oraNod maybe we can add option to docs now and implement it in the 2.0.0 milestone later

Reply:
https://github.com/infinispan/infinispan-operator/issues/435

Reply:
@dmvolod I think first comes the implementation then comes the doc